doc/source/serve/model-multiplexing.md

```diff
@@ -0,0 +1,68 @@
+# Model Multiplexing
+
+This section helps you understand how to write multiplexed deployment by using new introduced API `serve.multiplexed` and `serve.get_multiplexed_model_id`.
+
+This is experimental feature and the API may change in the future, welcome to try it out and give us feedback!
+
+## Why do we need model multiplexing?
+
+Model multiplexing is a technique to serve multiple models in a single replica, traffic will be routed to the corresponding model based on the request header.
```

Comment:
Can we change this to - "to serve multiple models with a pool of replicas to optimize cost. This is useful in cases where you might have many models with the same shape but different weights that are sparsely invoked. If any replica for the deployment has the model loaded, incoming traffic for that model (based on request header) will automatically be routed to that replica avoiding unnecessary load time."